nni/common/graph_utils.py

```diff
@@ -71,7 +71,11 @@ def __init__(self, model=None, dummy_input=None, traced_model=None):
     def _trace(self, model, dummy_input):
         training = model.training
         model.eval()
-        self.trace = torch.jit.trace(model, dummy_input)
+        kw_args = {}
+        if torch.__version__ >= '1.6.0':
+            # only pytorch with version greater than 1.6.0 has the strict option
+            kw_args['strict'] = False
+        self.trace = torch.jit.trace(model, dummy_input, **kw_args)
```

Comment:
does `strict=False` mean the same behavior as the jit.trace in the versions before 1.6.0?

Reply:
strict (bool, optional) – run the tracer in a strict mode or not (default: True). Only turn this off when you want the tracer to record your mutable container types (currently list/dict) and you are sure that the container you are using in your problem is a constant structure and does not get used as control flow (if, for) conditions.

The `strict` parameter enables the list of tensors and dict of tensors which is commonly used in detection model, in the version older than 1.6.0, it cannot support the list/dict as the input.